.circleci/config.yml

```diff
@@ -222,3 +265,13 @@ workflows:
           filters:
             branches:
               only: dev
+  nightly-integration-tests:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only: dev
+    jobs:
+      - integration-tests-alpha
+      - integration-tests-staging
```

Comment:
I had a thought... what if we run this on every merge to dev? I believe the goal is to test the combination of bleeding-edge UI and stable server prior to deploying UI changes. Performing this check on every dev merge would cover us for on-demand UI releases as well as daily scheduled releases.